tests/lib/rules/block-spacing.js

```diff
@@ -21,48 +21,48 @@ const ruleTester = new RuleTester();
 ruleTester.run("block-spacing", rule, {
     valid: [
 
-        // default/always
-        { code: "{ foo(); }", options: ["always"] },
-        "{ foo(); }",
-        "{ foo();\n}",
-        "{\nfoo(); }",
-        "{\r\nfoo();\r\n}",
-        "if (a) { foo(); }",
-        "if (a) {} else { foo(); }",
-        "switch (a) {}",
-        "switch (a) { case 0: foo(); }",
-        "while (a) { foo(); }",
-        "do { foo(); } while (a);",
-        "for (;;) { foo(); }",
-        "for (var a in b) { foo(); }",
-        { code: "for (var a of b) { foo(); }", parserOptions: { ecmaVersion: 6 } },
-        "try { foo(); } catch (e) { foo(); }",
-        "function foo() { bar(); }",
-        "(function() { bar(); });",
-        { code: "(() => { bar(); });", parserOptions: { ecmaVersion: 6 } },
-        "if (a) { /* comment */ foo(); /* comment */ }",
-        "if (a) { //comment\n foo(); }",
+        // // default/always
+        // { code: "{ foo(); }", options: ["always"] },
+        // "{ foo(); }",
+        // "{ foo();\n}",
```

Comment:
It seems that all `valid` tests are commented out.

Reply:
ohh sorry, I must have forgot to un-comment them.

Reply:
done